### PR TITLE
Add dev environment

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/developer/debugging.rst [hunk 1]
```diff
@@ -5,14 +5,31 @@ Debugging
...
 
-General
+Webpack
 -------
 
-First you should add ``?debug`` in the application URL to have all the
-JavaScript and stylesheets in separated non-minified files.
+To have all the JavaScript and stylesheets in separated non-minified files.
+
+Add in your makefile ``<user>.mk`` (Each devlopper should have a different port, e.g.: 8081):
```
Comment (jwkaltz) [inline doc/developer/debugging.rst]: devlopper -> developer

File: doc/developer/debugging.rst [hunk 2]
```diff
@@ -5,14 +5,31 @@ Debugging
...
+
+Browser
+-------
 
 Using a browser-integrated debugging tool usually available with the ``F12`` key.
 
+Pyramid
+-------
+
 If the ``pyramid_debugtoolbar`` is enabled the error is directly returned
```
Comment (jwkaltz) [inline doc/developer/debugging.rst]: "is enabled" -> "is enabled,"
"is directly returned" -> "is directly shown"

File: doc/developer/debugging.rst [hunk 3]
```diff
@@ -5,14 +5,31 @@ Debugging
 
 The goal of this document is to give some troubleshooting tips.
 
-General
+Webpack
 -------
 
-First you should add ``?debug`` in the application URL to have all the
-JavaScript and stylesheets in separated non-minified files.
+To have all the JavaScript and stylesheets in separated non-minified files.
```
Comment (jwkaltz) [inline doc/developer/debugging.rst]: I do not understand the connection between the header "Webpack", this sentence ("To have all the JavaScript...") and the following instructions.
Is there something missing?

Comment (sbrunner) [reply doc/developer/debugging.rst]: It's Webpack who minify the js and the css, and we use webpack-dev-derver to don't have this minification...

Comment (jwkaltz) [reply doc/developer/debugging.rst]: So we could say something like:
"For debugging purposes, it is better to have all the JavaScript and stylesheets in separated, non-minified files. To achieve this, you need to disable Webpack's minification; you can achieve this as follows.

What do you think?

Comment (sbrunner) [reply doc/developer/debugging.rst]: I just updated with a variant of your proposal :-)

**Pull request merged**